### PR TITLE
Build new /download/iot/intel-iotg page

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Build new page at `/download/iot/intel-iotg`
- Add to secondary nav 

## QA

- View page at: https://ubuntu-com-10332.demos.haus/download/iot/intel-iotg
- Check page against [copy doc](https://docs.google.com/document/d/1hxKzPs6e6JQaa9tBqfZ55GAXrVnEryEWBja8HlXy-lc/edit#)
- Note: the TM and R symbols are only used on the first and most prominent mentions of the mark word

## Issue / Card

Fixes [#4406](https://github.com/canonical-web-and-design/web-squad/issues/4406)

## Screenshots

Screenshot 2021-09-09 at 09 57 11

### CODE REVIEW

**Comment by bethcollins92** [comment]
@pmahnke could you code and qa review this please? I've kept the R and TM's on the titles even though it's the second time they're mentioned as that's the most prominent

File: templates/download/iot/intel-iotg.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      </li>
+    </ul>
+  </div>
+</section>
+<section class="p-strip">
+  <div class="row">
+    <div class="col-8">
+      <h3>Learn more about Intel Atom&reg; X6000E Series and Intel&reg; Pentium&reg; and Celeron&reg; N and J Series Processors</h3>
+      <p>To support the next generation of IoT edge devices, Intel has developed a new line of processors enhanced for IoT—The Intel Atom x6000E Series, Intel Pentium, and Intel Celeron N and J Series processors (formerly known as Elkhart Lake - EHL). These processors build on new levels of CPU, and graphics performances with integrated IoT features, real-time performance, manageability, security, and functional safety.</p>
+      <p><a class="p-link--external" href="https://www.intel.com/content/www/us/en/products/docs/processors/embedded/enhanced-for-iot-platform-brief.html">Read the platform brief</a></p>
```
Comment (anthonydillon) [inline templates/download/iot/intel-iotg.html:50]: Can you remove the `p` here to remove the margin bottom please? Matching it with the row below.

File: templates/download/iot/intel-iotg.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
+{% extends "download/_base_download.html" %}
+
+{% block title %}Download Ubuntu for Intel IoT platforms | Download{% endblock %}
```
Comment (anthonydillon) [inline templates/download/iot/intel-iotg.html:3]: Could you create a social banner for this page using https://tools.demos.haus/. You can see what this page looks like on social media here: https://www.opengraph.xyz/url/https%3A%2F%2Fubuntu-com-10332.demos.haus%2Fdownload%2Fiot%2Fintel-iotg/

File: templates/download/iot/intel-iotg.html [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+{% block meta_description %}​​Canonical and Intel have partnered to provide Ubuntu Desktop and Ubuntu Core on Intel's IoT platforms: Atom&reg; X6000E Series, Intel&reg; Pentium&reg; and Celeron&reg; N and J Series and 11th Gen Intel&reg; Core&trade; processors.{% endblock meta_description %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1hxKzPs6e6JQaa9tBqfZ55GAXrVnEryEWBja8HlXy-lc/edit{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--suru-topped">
+  <div class="row u-vertically-center">
+    <div class="col-7">
+      <h1>Download Ubuntu for Intel IoT platforms</h1>
+      <p class="p-heading--4">Ubuntu - now optimized for Intel's IOTG platforms</p>
```
Comment (anthonydillon) [inline templates/download/iot/intel-iotg.html:13]:
```suggestion
      <p class="p-heading--4">Ubuntu &mdash; now optimized for Intel's IOTG platforms</p>
```

File: templates/download/iot/intel-iotg.html [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+      <p>Ubuntu Core 20 and Ubuntu Desktop 20.04 based images for Elkhart Lake and Tiger Lake platforms will be released on October 14th and are currently available as a private alpha.</p>
+      <a class="p-button--positive" href="mailto:sales-iotg@canonical.com&subject=Ubuntu+Core+20+private+alpha+for+Elkhart+Lake+and+Tiger+Lake">Contact us if you'd like early access</a>
```
Comment (anthonydillon) [inline templates/download/iot/intel-iotg.html:15]: I hate this, it's an anti-pattern these days. [:information_source:](https://blog.seibert-media.com/2014/10/06/why-mailto-links-should-be-avoided-on-websites/)

File: templates/download/iot/intel-iotg.html [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+      <p><a class="p-link--external" href="https://www.intel.com/content/www/us/en/products/docs/processors/embedded/enhanced-for-iot-platform-brief.html">Read the platform brief</a></p>
+    </div>
+  </div>
+  <div class="u-fixed-width">
+    <hr class="p-separator">
+  </div>
+  <div class="row">
+    <div class="col-8">
+      <h3>Learn more about Intel 11th Gen Intel Core&trade; processors</h3>
+      <p>11th Gen Intel Core processors (formerly known as Tiger Lake - TGL) deliver a balance of performance and responsiveness in a low-power platform built on our third-generation 10nm process technology. Engineered to deliver for IoT markets, these processors can support low-latency and time-sensitive applications, and have the power to run multiple workloads including AI and deep learning applications on a single platform.</p>
```
Comment (anthonydillon) [inline templates/download/iot/intel-iotg.html:59]:
```suggestion
      <p>11th Gen Intel Core processors (formerly known as Tiger Lake &mdash; TGL) deliver a balance of performance and responsiveness in a low-power platform built on our third-generation 10nm process technology. Engineered to deliver for IoT markets, these processors can support low-latency and time-sensitive applications, and have the power to run multiple workloads including AI and deep learning applications on a single platform.</p>
```

File: templates/download/iot/intel-iotg.html [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+        Intel 11th Gen Intel Core&trade; processors
+      </li>
+    </ul>
+  </div>
+</section>
+<section class="p-strip">
+  <div class="row">
+    <div class="col-8">
+      <h3>Learn more about Intel Atom&reg; X6000E Series and Intel&reg; Pentium&reg; and Celeron&reg; N and J Series Processors</h3>
+      <p>To support the next generation of IoT edge devices, Intel has developed a new line of processors enhanced for IoT—The Intel Atom x6000E Series, Intel Pentium, and Intel Celeron N and J Series processors (formerly known as Elkhart Lake - EHL). These processors build on new levels of CPU, and graphics performances with integrated IoT features, real-time performance, manageability, security, and functional safety.</p>
```
Comment (anthonydillon) [inline templates/download/iot/intel-iotg.html:49]:
```suggestion
      <p>To support the next generation of IoT edge devices, Intel has developed a new line of processors enhanced for IoT—The Intel Atom x6000E Series, Intel Pentium, and Intel Celeron N and J Series processors (formerly known as Elkhart Lake &mdash; EHL). These processors build on new levels of CPU, and graphics performances with integrated IoT features, real-time performance, manageability, security, and functional safety.</p>
```

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Looking good Beth, just a few small requests in the code.

**Comment by long-chung** [comment]
Looks good. Just one question; should the ® symbol be unicode or superscript?

File: templates/download/iot/intel-iotg.html (hunk 4 above)
Comment (bethcollins92) [reply templates/download/iot/intel-iotg.html:15]: I agree - Peter added this yesterday I presumed it was a placeholder due to the urgency of this page. (The comment you left on the copy doc has been resolved by David, though I'm not sure why as it hasn't been answered)

**Comment by bethcollins92** [comment]
@anthonydillon Made requested changes - but from the copy doc it seems we are still waiting for [this](https://docs.google.com/document/d/1hxKzPs6e6JQaa9tBqfZ55GAXrVnEryEWBja8HlXy-lc/edit?disco=AAAAOMf6YYE) and the form but if that's gonna take a few days maybe we will need to merge it as is

File: templates/download/iot/intel-iotg.html (hunk 2 above)
Comment (anthonydillon) [reply templates/download/iot/intel-iotg.html:3]: Nice :+1:

**Comment by anthonydillon** [comment]
> @anthonydillon Made requested changes - but from the copy doc it seems we are still waiting for [this](https://docs.google.com/document/d/1hxKzPs6e6JQaa9tBqfZ55GAXrVnEryEWBja8HlXy-lc/edit?disco=AAAAOMf6YYE) and the form but if that's gonna take a few days maybe we will need to merge it as is

I guess, I also suggested we could:
> I would suggest it would be better to say "Send an email to sales-iotg {@} canonical.com with the subject line of 'Ubuntu Core 20 private alpha for Elkhart Lake and Tiger Lake' to register interest in early access."  Even it the button acted as an accordion trigger for that information

But if this is under a tight deadline I guess we can come back to it.

**Review by anthonydillon — APPROVED** [review]
Code and QA :+1:

**Comment by bethcollins92** [comment]
@anthonydillon I've updated it with an information notification let me know what you think

**Comment by anthonydillon** [comment]
I think it's better and good for now. Thanks for the update. Could you update the copy doc too?

**Comment by bethcollins92** [comment]
Sam ok'd merging without Pdm +1

**Pull request merged**